### PR TITLE
Latest strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -13,25 +13,24 @@
...
     <string name="delete_all_actions_message_q">Are you sure you want to irrevocably delete %d quick actions?</string>
     <string name="shared_string_delete_all_q">Delete all?</string>
-    <string name="export_import_quick_actions_with_profiles_promo">You can Export or Import quick actions with application profiles.</string>
-    <string name="tiles_storage_descr">Choose how downloaded tiles will be stored.</string>
-    <string name="expire_time_descr">Expiration time in minutes. Cached tiles will be reloaded after specified time.
-        Leave this field empty to never refresh tiles for this source.
+    <string name="export_import_quick_actions_with_profiles_promo">You can export or import quick actions with app profiles.</string>
+    <string name="tiles_storage_descr">Choose how to store downloaded tiles.</string>
+    <string name="expire_time_descr">Temporarily downloaded tiles will be redownloaded after a specified amount of minutes.
+        Leaft empty tiles will never be redownloaded for this source.
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:27]:
```suggestion
        Left empty tiles will never be redownloaded for this source.
```

File: OsmAnd/res/values/strings.xml [hunk 2]
```diff
@@ -13,25 +13,24 @@
...
     <string name="screen_timeout">Screen timeout</string>
     <string name="delete_all_actions_message_q">Are you sure you want to irrevocably delete %d quick actions?</string>
     <string name="shared_string_delete_all_q">Delete all?</string>
-    <string name="export_import_quick_actions_with_profiles_promo">You can Export or Import quick actions with application profiles.</string>
-    <string name="tiles_storage_descr">Choose how downloaded tiles will be stored.</string>
-    <string name="expire_time_descr">Expiration time in minutes. Cached tiles will be reloaded after specified time.
-        Leave this field empty to never refresh tiles for this source.
+    <string name="export_import_quick_actions_with_profiles_promo">You can export or import quick actions with app profiles.</string>
+    <string name="tiles_storage_descr">Choose how to store downloaded tiles.</string>
+    <string name="expire_time_descr">Temporarily downloaded tiles will be redownloaded after a specified amount of minutes.
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:26]: Not sure about this.

File: OsmAnd/res/values/strings.xml [hunk 3]
```diff
@@ -41,23 +40,23 @@
...
-    <string name="turn_screen_on_power_button_disabled">Disabled. Requires \'Keep screen on\' under \'Timeout after wake-up\'.</string>
+    <string name="turn_screen_on_power_button_descr">Pressing the power button on your device turns the screen on with OsmAnd on top of the lock screen.</string>
+    <string name="turn_screen_on_power_button_disabled">Off. Requires \'Keep screen on\' under \'Timeout after wake-up\'.</string>
     <string name="turn_screen_on_navigation_instructions">Navigation instructions</string>
     <string name="turn_screen_on_navigation_instructions_descr">Each navigation instruction will turn the screen on.</string>
     <string name="turn_screen_on_descr">Select screen wake-up options (make sure OsmAnd is in the foreground when the device is being locked):</string>
     <!--string name="turn_screen_on_options_descr">Screen wake-up options:</string-->
-    <string name="system_screen_timeout">Use system screen timeout</string>
-    <string name="system_screen_timeout_descr">Turns the screen off after the system\'s screen timeout.</string>
+    <string name="system_screen_timeout">Let the system decide screen timeout</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:52]:
```suggestion
    <string name="system_screen_timeout">Let the system decide how long the screen should stay on</string>
```

File: OsmAnd/res/values/strings.xml [hunk 4]
```diff
@@ -2380,8 +2379,8 @@
     <string name="driving_region_japan">Japan</string>
     <string name="driving_region_us">United States</string>
     <string name="driving_region_canada">Canada</string>
-    <string name="driving_region_europe_asia">Europe, Asia, Latin America, &amp; similar</string>
-    <string name="driving_region_uk">UK, India, &amp; similar</string>
+    <string name="driving_region_europe_asia">Europe, Asia, Latin America, and similar</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:2382]: The ampersand use is all over the place

File: OsmAnd/res/values/strings.xml [hunk 5]
```diff
@@ -2431,7 +2430,7 @@
     <string name="distance_measurement_finish_subtrack">Begin a new subtrack</string>
     <string name="distance_measurement_clear_route">Clear all points</string>
     <string name="distance_measurement_load_gpx">Open existing GPX file</string>
-    <string name="wait_current_task_finished">Please wait until current task is finished</string>
+    <string name="wait_current_task_finished">Please wait for the current to finish</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:2433]:
```suggestion
    <string name="wait_current_task_finished">Please wait until current task is finished</string>
```

File: OsmAnd/res/values/strings.xml [hunk 6]
```diff
@@ -13,25 +13,24 @@
...
     <string name="delete_all_actions_message_q">Are you sure you want to irrevocably delete %d quick actions?</string>
     <string name="shared_string_delete_all_q">Delete all?</string>
-    <string name="export_import_quick_actions_with_profiles_promo">You can Export or Import quick actions with application profiles.</string>
-    <string name="tiles_storage_descr">Choose how downloaded tiles will be stored.</string>
-    <string name="expire_time_descr">Expiration time in minutes. Cached tiles will be reloaded after specified time.
-        Leave this field empty to never refresh tiles for this source.
+    <string name="export_import_quick_actions_with_profiles_promo">You can export or import quick actions with app profiles.</string>
+    <string name="tiles_storage_descr">Choose how to store downloaded tiles.</string>
+    <string name="expire_time_descr">Temporarily downloaded tiles will be redownloaded after a specified amount of minutes.
+        Left empty tiles will never be redownloaded for this source.
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:27]:
```suggestion
        Left empty tiles will never be re-downloaded for this source.
```

File: OsmAnd/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:26]:
```suggestion
    <string name="expire_time_descr">Temporarily downloaded tiles will be re-downloaded after a specified amount of minutes.
```

File: OsmAnd/res/values/strings.xml [hunk 7]
```diff
@@ -41,23 +40,23 @@
     <string name="pseudo_mercator_projection">Pseudo-Mercator projection</string>
     <string name="keep_screen_off">Keep screen off</string>
     <string name="keep_screen_on">Keep screen on</string>
-    <string name="turn_screen_on_wake_time_descr">Select the screen timeout after waking up. (\"%1$s\" applies no timeout.)</string>
+    <string name="turn_screen_on_wake_time_descr">Select how long the screen should be kept on. (\"%1$s\" keeps the screen from turning off.)</string>
```
Comment (sonora) [inline OsmAnd/res/values/strings.xml:43]: It's tricky, this timeout is not the regular timeout, i really only applies after a wake-up.

**Pull request merged**